packages/generators/utils/plugins.ts

```diff
@@ -6,6 +7,44 @@
  * that consists of terser-webpack-plugin
  */
 
-export default function(): string[] {
+export default function (): string[] {
 	return ["new TerserPlugin()"];
 }
+
+/**
+ *
+ * Replaces the string with a substring at the given index
+ * https://gist.github.com/efenacigiray/9367920
+ *
+ * @param	{String} str - string to be modified
+ * @param	{Number} index - index to replace from
+ * @param	{String} replace - string to replace starting from index
+ *
+ * @returns	{String} string - The newly mutated string
+ *
+ */
+
+export const replaceAt = (str: string, index: number, replace: string) : string => {
+	return str.substring(0, index) + replace + str.substring(index + 1);
+}
+
+
+/**
+ *
+ * Generate a webpack standard webpack plugin name from the plugin name from the Answer
+ *
+ * @param	{String} rawPluginName - plugin name from answer
+ *
+ * @returns	{String} string - the webpack standard plugin name
+ *
+ */
+
+
+export const generatePluginName = (rawPluginName: string): string => {
+	let myPluginNameArray : string[];
+	myPluginNameArray = rawPluginName.split("-");
+	for (let i = 0; i < myPluginNameArray.length && myPluginNameArray.length > 1 ; i++) {
```

Comment:
Great. So the next step now is to extract the length of the array into a variable so that GC will be better. The v8 compiler engine by Google recommends this because the optimizations at parse time will be improved, so there's no reason not to change this. 

You simply declare `const pluginArrLength = myPlufinName.length` and use that to compare

https://stackoverflow.com/questions/6973942/is-optimizing-javascript-for-loops-really-necessary

Reply:
done @evenstensberg.